pulsar/function/descriptor.json

```diff
@@ -0,0 +1,21 @@
+{
+	"name": "pulsar-function-trigger",
+	"type": "flogo:trigger",
+	"version": "0.0.1",
+	"title": "Apache Pulsar Trigger Function",
+	"description": "A simple pulsar function which executes Flogo.",
+	"settings": [
+	
+	],
+	"output": [
+		{
+			"name": "message",
+			"type": "string",
+			"allowed": null,
+			"required": false
+		}
+	],
+	"handler": {
+		
+	}
+}
```

Comment:
We will also need to support a reply field, as pulsar functions support function replays to topics. Basically the framework will publish the function response to a specific topic... Pretty handy

Reply:
See this as an example of how to return the output: https://github.com/apache/pulsar/blob/master/pulsar-function-go/examples/outputFunc/outputFunc.go

Reply:
It does reply by default but I'm not sure how to add it to descriptor.

Reply:
Doesn’t the flow need to map data back to the trigger, otherwise how would we know what’s actually returned? I thought that was done with the reply mapping?